packages/web-frontend/src/utils/getUniqueViewId.js

```diff
@@ -6,8 +6,5 @@
 export const getUniqueViewId = ({ organisationUnitCode, dashboardGroupId, viewId }) =>
   [organisationUnitCode, dashboardGroupId, viewId].join('___');
 
-export const getViewIdFromInfoViewKey = infoViewKey => {
-  const infoViewKeyParams = infoViewKey.split('___');
-  if (infoViewKeyParams && infoViewKeyParams.length === 3) return infoViewKeyParams[2];
-  return null;
-};
+export const getViewIdFromInfoViewKey = infoViewKey =>
+  infoViewKey ? infoViewKey.split('___')[2] : null;
```

Comment:
Not exactly what was requested just FYI

Reply:
This should be fine under the conditions that

* if `infoViewKey` exists, it will be in a valid form including all components, or
* `undefined` and `null` are both compatible "falsy" values for clients of this function

Reply:
I think that the second one is true, but I have made an edit just in case